free-programming-books-it.md

```diff
@@ -47,6 +47,9 @@
 ### BASH
 
 * [Guida avanzata per la bash](http://www.dmi.unict.it/diraimondo/web/wp-content/uploads/classes/so/mirror-stuff/abs-guide.pdf) (PDF)
+* [Guida per principianti bash](http://codex.altervista.org/guidabash/guidabash_1_11.pdf) (PDF)
+* [PC Assembly Language](http://drpaulcarter.com/pcasm/) - Paul A. Carter
+* [Programmazione della shell Bash](http://www.aquilante.net/doc/bash_programming.pdf) (PDF)
```

Comment:
the author say: "
Questa guida può essere liberamente fotocopiata o riprodotta con ogni altro mezzo, purché sia distribuita integralmente e gratuitamente, senza scopo di lucro. Quanto riportato nella guida può essere citato liberamente, purché ciò avvenga nel rispetto del copyright che rimane di proprietà dell'autore. Il presente documento è disponibile su Internet all'indirizzo http://www.aquilante.net/bash/. "